crates/bevy_pbr/src/meshlet/visibility_buffer_raster_node.rs

```diff
@@ -94,7 +102,7 @@ impl Node for MeshletVisibilityBufferRasterPassNode {
             .fetch_and(false, Ordering::SeqCst);
 
         let thread_per_cluster_workgroups =
-            (meshlet_view_resources.scene_meshlet_count.div_ceil(128) as f32)
+            (meshlet_view_resources.scene_cluster_count.div_ceil(128) as f32)
                 .cbrt()
```

Comment:
This is existing code, but each workgroup processes 128 clusters, so we need `x = ceil(N / 128)` workgroups total. Of course, since we want one dispatch over every single cluster (millions across all instances/LODs) in the scene, we'd quickly hit the ~65,536 per-dispatch-dimension limit if we tried to dispatch 3 million workgroups in only the x dimension. Can't do a 1d dispatch.

Using the ceil of the cuberoot lets us spread the dispatch across all 3 x/y/z dimensions to get around the limit. In the shader, global_workgroup_id is converted back to a 1d index. 

This is the same trick as the remap_1d_to_2d_dispatch shader, which does the same thing on the GPU for converting a 1d indirect dispatch to a 2d dispatch (3d is unlikely to ever be necessary).

This is going to be irrelevant once I switch to persistent threads-style culling of clusters, as then we just need (1?) a low, fixed number of workgroups to process the scene no matter the cluster count.

The rename from scene_meshlet_count to scene_cluster_count is just a rename that was missed when making the terminology more consistent in past PRs, it's not functionally different.